docker/localnet/container-env/observer-0.env

```diff
@@ -0,0 +1,11 @@
+# Environment variables used by the container
+
+# General
+CHEQD_NODED_MONIKER="observer-0"
+CHEQD_NODED_RPC_LADDR="tcp://0.0.0.0:26657"
```

Comment:
Don't you also need to set P2P_LADDR (or is that by default)?

Reply:
For p2p it's 0.0.0.0 by default while for RPC it's limited to localhost